pt-BR/association_basics.md

```diff
@@ -2455,62 +2455,65 @@ Extensions can refer to the internals of the association proxy using these three
-the `color` and `price` fields and some methods for all of them, but having some
-specific behavior for each, and separated controllers too.
+As vezes é desejável compartilhar atributos e comportamento entre _models_.
+Vamos dizer que temos _models_ `Car`, `Motorcycle` e `Bicycle`. Queremos
+compartilhar os atributos de `color` e `price` e também alguns métodos para
+estes atributos, mas ainda mantendo comportamentos específicos para cada um
+deles, incluindo _controllers_ separados.
 
-Rails makes this quite easy. First, let's generate the base Vehicle model:
+Rails deixa isso bem fácil. Primeiro, vamos gerar o _model_ de base, `Vehicle`:
 
 ```bash
 $ rails generate model vehicle type:string color:string price:decimal{10.2}
 ```
 
-Did you note we are adding a "type" field? Since all models will be saved in a
-single database table, Rails will save in this column the name of the model that
-is being saved. In our example, this can be "Car", "Motorcycle" or "Bicycle."
-STI won't work without a "type" field in the table.
+Você notou que estamos adicionando um atributo `type`? Dado que todos os _models_
+vão ser armazenados em uma única tabela, Rails vai armazenar o nome do _model_
+nesse atributo. No nosso exemplo, as possibilidades são `Car`, `Motorcycle` ou
+`Bicycle`. STI não funciona sem um atributo `type` na tabela.
 
-Next, we will generate the three models that inherit from Vehicle. For this,
-we can use the `--parent=PARENT` option, which will generate a model that
-inherits from the specified parent and without equivalent migration (since the
-table already exists).
+Em seguida, vamos gerar os três _models_ que herdam de `Vehicle`. Para isso podemos
+usar a opção `--parent=PARENT` que vai gerar um _model_ que herda do "_parent_"
+especificado e sem uma _migration_ equivalente (dado que a tabela já existe).
 
-For example, to generate the Car model:
+Por exemplo, para gerar o _model_ `Car`:
 
 ```bash
 $ rails generate model car --parent=Vehicle
 ```
 
-The generated model will look like this:
+O _model_ gerado vai parecer com:
 
 ```ruby
 class Car < Vehicle
 end
 ```
 
-This means that all behavior added to Vehicle is available for Car too, as
-associations, public methods, etc.
+Isso significa que todo o comportamento adicionado à classe `Vehicle` estará
+disponível também na classe `Car`, incluindo as associações, métodos publicos,
+etc.
 
-Creating a car will save it in the `vehicles` table with "Car" as the `type` field:
+Criar um carro vai armazená-lo na tabela `vehicles` e popular o atributo `type`
+com o valor "_Car_":
 
 ```ruby
 Car.create(color: 'Red', price: 10000)
 ```
 
-will generate the following SQL:
+vai gerar o seguinte SQL:
 
 ```sql
 INSERT INTO "vehicles" ("type", "color", "price") VALUES ('Car', 'Red', 10000)
 ```
 
-Querying car records will just search for vehicles that are cars:
+Em uma busca por registros de carros vai simplesmente buscar veiculos que são
```

Comment:
Aqui, acredito que podemos deixar explícito o sujeito no meio da frase, para dar mais contexto. Pensei em algo como:

`Em uma busca por registros de carros, o sql (ou query, ou consulta, não sei ao certo qual o mais adequado) vai simplesmente buscar veículos que são do tipo _Car_`

O que você acha @lastgabs ? Será que vai ficar redundante?
@campuscode/campus-code-team podem opinar aqui também? :smiley:

Reply:
Eu acho que gosto de colocar a *query* no início da frase, por exemplo:

> A *query* (consulta) por registros de carros vai simplesmente buscar veículos que são... 

Deixei 'consulta' entre parenteses, mas pode ser dispensado se o termo *query* já estiver de alguma forma consolidado como sinônimo de "busca" ou algo semelhante. Como é uma palavra que, apesar de ter tradução, é também um termo técnico utilizado em inglês mesmo na maior parte do tempo, acho que é bom reforçar... Mas isso é um pouco "nitpicking"... não vejo nada de errado do jeito que está... 

:)